CHANGELOG.md

```diff
@@ -43,6 +43,7 @@
   - `try_to_binary`
 - Added support for specifying a schema string (including implicit struct syntax) when calling `DataFrame.create_dataframe`.
 - Added support for `DataFrameWriter.insert_into/insertInto`. This method also supports local testing mode.
+- Added support for `seed` argument in `DataFrame.stat.sample_by`. Note that it only supports a `Table` object, and will be ignored for a `DataFrame` object.
```

Comment:
How about logging a warning when the DataFrame isn't a Table?

Reply:
Yea it's already added